starter-code/apple-pie-starter-code/css/main.css

```diff
@@ -0,0 +1,41 @@
+body {
+    font-family: "Open Sans", sans-serif;
+}
+
+header {
+    background-image: url("../images/apple-pie.jpg");
+    background-size: cover;
+    height: 860px;;
+}
+
+div h1 {
+    color: white;
+    text-align: center;
+    font-size: 50px;
+    padding-left: 25%;
```

Comment:
En esto caso sirve pero de cara al futuro para responsive tenéis mejores alternativas! Aún así habéis usado flex para otras cosas, muy bien por proponer alternativas 👍